parcelable/src/commonMain/kotlin/com/arkivanov/essenty/parcelable/RawValue.kt

```diff
@@ -0,0 +1,11 @@
+package com.arkivanov.essenty.parcelable
+
+/**
+ * A multiplatform alias for `kotlinx.parcelize.RawValue` from the `kotlin-parcelize` Gradle plugin.
+ * Make sure you have the `kotlin-parcelize` Gradle plugin applied.
+ */
+@OptIn(ExperimentalMultiplatform::class)
+@OptionalExpectation
+@Target(AnnotationTarget.TYPE)
+@Retention(AnnotationRetention.BINARY)
+expect annotation class RawValue()
```

Comment:
I would avoid adding `RawValue` for now, because I'm trying to keep the `parcelable` module platform agnostic. E.g. currently it's possible to [write a similar compiler plugin](https://github.com/arkivanov/kotlin-parcelize-darwin) for Darwin (Apple) targets. `RawValue` uses Java serialization and it's not clear how it could be implemented for other platforms.

Users may add their own expect/actual if they really need it, so shouldn't be a deal breaker.

Reply:
Sure, no problem, will remove it.